src/containers/RecentData.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types"
+import Data from "../components/data";
+import statsClient from "panoptes-client/lib/stats-client";
```

Comment:
Possibly discussed in Slack, but not sure if we want to use the old stats API (imported here, I think [zoo-event-stats](https://github.com/zooniverse/zoo-event-stats)) or the new stats API (https://github.com/zooniverse/zoo-stats-api-graphql)?

I think we'll be using new in new frontend - https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/stores/YourStats.js#L8.

Reply:
Gotcha. Didn't know we were moving away from the old API. I'll see if I can move it over, especially since I'll eventually be using the new one with daily country counts.